### ISSUE TITLE
[pni] cross page: missing `<main>` element

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#4640]
The `<main>` element is relied on by accessibility tools to directly navigate to the start of the main content... we don't have one =)

**Comment by Pomax** [comment]
actually a dupe of https://github.com/mozilla/foundation.mozilla.org/issues/4641 really
